README.md

```diff
@@ -176,6 +174,25 @@ type Profile = {
 - `additionalLogoutParams`: dictionary of additional query params to add to 'logout' requests
 - `logoutCallbackUrl`: The value with which to populate the `Location` attribute in the `SingleLogoutService` elements in the generated service provider metadata.
 
+- **Extensions**
+- `extensions`: Optional, The extension provides a more flexible structure for expressing which combination of Attributes are requested by service providers in comparison to the existing mechanisms, [More about extensions](https://www.oasis-open.org/committees/download.php/55790/Connectis%20_protocol_extension_draft.pdf). It accept fully customize [XMLBuilder](https://www.npmjs.com/package/xmlbuilder) type. There are many possible value for `extensions` element. You can use [XMLBuilder](https://www.npmjs.com/package/xmlbuilder) to request attributes/data as per your need in SAML Request.
```

Comment:
The grammar here is a little off, but it is still understandable. We can fix it all at once later.

Reply:
We might want to change the reference to the approved spec, or at least add it. The PDF is a good reference, but is listed as a draft. Here is the official spec that we want to support: https://docs.oasis-open.org/security/saml-protoc-req-attr-req/v1.0/saml-protoc-req-attr-req-v1.0.html

Reply:
@cjbarth first of all thank you for review and sorry about my English. I agreed to add official docs link instead of PDF. I'll update it.

Reply:
Don't worry about your level of English. Learning a second language is not easy. I'll fix it up for you as a last item before we land this.

Reply:
thanks mate :)